UniAtHome/UniAtHome.BLL/DTOs/Auth/LoginResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System.Collections.Generic;
+
+namespace UniAtHome.BLL.DTOs.Auth
+{
+    public sealed class LoginResponse : ErrorProneOperationResponse
```

Comment:
Please, put responses entities into Web.API project

Reply:
Why? Responses are used only as the results of services' requests. They ain't designed to be used the other way. In addition BLL should not depend on the WEB Api project (it's even impossible to make)